org.kde.Ikona.json

```diff
@@ -0,0 +1,43 @@
+{
+    "id": "org.kde.Ikona",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "ikona",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
this won't work. There is not network access.

Reply:
```
Error: Can't specify build-args in sandboxed build
```

Reply:
Rust dependencies are downloaded at compile time, so how can I achieve that here? I'll have to look at manifests of other Rust applications on Flathub.

Reply:
like python or node

https://github.com/flatpak/flatpak-builder-tools

Reply:
Thanks for the pointer :)